tests/python_tests/conftest.py

```diff
@@ -1,28 +1,58 @@
+import os
 import pytest
+import shutil
+import logging
+from utils.constants import get_ov_cache_models_dir
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup_and_teardown():
```

Comment:
Can you merge two `conftest.py` instead?

Reply:
In order to reuse `conftest.py` download and convert approaches should be aligned between API and sample tests. I propose to address it in another PR.